doc/guides/getting_started/src/e2e_regression_setup.md

```diff
@@ -0,0 +1,58 @@
+# ROM E2E Regressions setup
+This guide will help you to setup an environment to run the ROM E2E tests in an Ubuntu 20.04 machine.
+
+## Introduction
+The [ROM](https://opentitan.org/book/doc/security/specs/secure_boot/index.html?highlight=ROM#rom-rom) is first boot stage of secure boot and it is placed in the ROM memory which means that it cannot be updated after manufacturing at all.
+The ROM E2E (End-to-End) tests validates the ROM features and can be used for regression tests.
+Each test is divided into components:
+ - **Opentitantool**, it is a Rust program that provide a common interface to interact with Opentitan. Opentitantool uses opentitanlib which provides the abstractions to communicate with Opentitan.
+ - **Host test** software, it is a Rust program built by bazel that actively interacts with the device software via UART, SPI or JTAG to perform the test and check whether it passed or not. To avoid code duplication the tests uses the `opentitanlib` to communicate with opentitan.
+ - **Device test** software, it is a C program built by bazel that run in the flash after the ROM and perform checking in the HW and/or memories to check the configurations performed by the ROM.
+ - **FPGA bitstream**, it is built by bazel which using vivado.
+ - **ROM Image**, it is a C program built by bazel which is the DUT (Device Under Test).
+ - **OTP image**, contains secure configurations read by the ROM to define workflows such as the lifecycle stage, so for some tests may be executed with several different OTP images to test all corner cases.
+  
+![Architecture](ROM_E2E_Tests.svg)
+
+All the existing tests are documented in the [ROM E2E Testplan](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson).
+## Hardware setup
+### FPGA
+The E2E ROM tests run in the CW310 FPGA board with the following configuration:
+ - The power supply connected to the J11 connector.
+ - A USB cable connecting the Host PC to the USB-C Data (J8) connector. 
+ - The USRDIP#0 (SW4 header) is set to 0.
+ - The rest of the USRDIPs are set to 1.
+ - The S3 switch (next to USB connector J9) is set to USB.
+ - The S2 switch (next to JTAG header) is set to +3.3V.
+ - Optionally, a USB cable connecting the host PC to the USB-C J9 connector for future tests that will test the USB block.
+  
+### JTAG adapter
+Some tests will use the JTAG interface to interact with Opentitan, as the CW310 doesn't have a embedded JTAG adapter, we can use an external adapter (Olimex ARM-USB-TINY-H JTAG) connected to the J13 header in the CW310.
+
+### Full hardware setup
+![Full hardware setup](CW310_setup.png)
```

Comment:
Just a point of clarification on the cabling shown in the image:
- The red USB cable connects to the SAM3U control chip on the board.
- The white USB cable connects to the user USB port and this port connects to the OT `usbdev` IP.

AFAIK, we currently do not have any tests that use the `usbdev` IP.  I believe such tests are under development at lowRISC.